courses/machine_learning/production_ml/3_streaming_data.ipynb

```diff
@@ -0,0 +1,390 @@
+{
```

Comment:
<p> </p><ul><li> move import to the beginning with all the other imports</li><li>I think you don't need to pass the PROJECT to the bigquery client. It will default to the project set in the environment</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/612/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>looks like you're right. I've removed that. </p>